### PR TITLE
[Docs] OR review

### PR DESCRIPTION
**Pull request by luarss**
According to the google [docs ](https://docs.google.com/document/d/1UQ-UbyEPy9UKQAtiVg6x9kPvOg-eeLFHYf_F6QHOhh0/edit)

Changes made:
- 1,5, 6, 7, 8, 11, 12, 13, 14, 15, 16, 17, 18

Changes in progress:
- 2, 3, 4

@vvbandeira @dralabeing @vijayank88

### CODE REVIEW

**Comment by dralabeing** [comment]
@luarss Is this ready for review?

**Comment by luarss** [comment]
Yes ready

File: docs/contrib/DeveloperGuide.md [hunk 1]
```diff
@@ -250,9 +250,9 @@ Tool namespaces are usually three-lettered lowercase letters.
 - Antenna check and diode insertion ([ant](../main/src/ant/README.md))
 - TritonRoute Detailed routing ([drt](../main/src/drt/README.md))
 - Metal fill insertion ([fin](../main/src/fin/README.md))
-- Design for Test ([dst](../main/src/dst/README.md))
+- Design for Test ([dft](../main/src/dft/README.md))
 - OpenRCX Parasitic Extraction ([rcx](../main/src/rcx/README.md))
-- OpenSTA timing/power report ([sta](../main/src/sta/README.md))
+- OpenSTA timing/power report ([sta](https://github.com/The-OpenROAD-Project/OpenSTA/blob/master/README.md)
 - Graphical User Interface ([gui](../main/src/gui/README.md))
 - Static IR analyser ([psm](../main/src/psm/README.md))
```
Comment (dralabeing) [inline docs/contrib/DeveloperGuide.md:257]: Change to analyzer

**Review by dralabeing — CHANGES_REQUESTED** [review]
Changes look good..except for the marked

File: docs/contrib/DeveloperGuide.md (hunk 1 above)
Comment (luarss) [reply docs/contrib/DeveloperGuide.md]: Fixed.

**Review by dralabeing — APPROVED** [review]

**Comment by luarss** [comment]
The remaining changes 2-4 will be addressed in the next PR, since it involves creation of some new files. @dralabeing @vvbandeira 

We can merge this first.

**Review by vvbandeira — APPROVED** [review]

**Pull request merged**